authorization/role/repository/identity_role_blackbox_test.go

```diff
@@ -178,6 +181,111 @@ func (s *identityRoleBlackBoxTest) TestFindIdentityRolesByResource() {
 	require.Equal(s.T(), identityRole.IdentityRoleID, identityRoles[0].IdentityRoleID)
 }
 
+func (s *identityRoleBlackBoxTest) TestFindIdentityRolesByIdentityAndResource() {
+	g := s.DBTestSuite.NewTestGraph()
+	newSpace := g.CreateSpace(g.ID("myspace"))
+
+	var createdIdentities []uuid.UUID
+
+	for i := 0; i <= 10; i++ {
+		user := g.CreateUser(g.ID("foo"))
```

Comment:
You can't give each created user the same ID

Reply:
In fact, I can't see where this identifier is even used - why is it being set?

Reply:
Well the method was looking for it, hence passed it. I suppose we are gonna wire up the passed argument somewhere later on?

Reply:
You don't need to set an ID if you don't intend to look up the object later on.  See [this](https://github.com/fabric8-services/fabric8-auth/blob/master/authorization/team/service/team_service_blackbox_test.go#L80) for an example.